cfpb/cfgov-refresh
# Support for special characters in tags!!!!!!!!!

## Context 
- See GHE #235 
- Previously, if a special character was a part of a tag (e.g. "Consumers' response"), the front-end would throw an uncaught JS error and multiselect would not work! Oh no.
- Now, this is fixed.

## Manually test (if you want to)
- Pull down branch & run `gulp scripts`
- Add a tag to a page under `data-research/research-reports` with special characters (such as an apostrophe) 
- Go to [http://localhost:8000/data-research/research-reports/](http://localhost:8000/data-research/research-reports/) and see that the multiselect in "Topics" in "Filter reports" still works, and displays the tag you entered correctly.
- Go to other pages and click on links from the topic tags or author tags and confirm things are working as expected.

## TODO
- Will update CHANGELOG once this is ready to get merged in

## Credits
- @contolini and I paired on this
- Thanks to @kurtrwall and his work here https://github.com/cfpb/cfgov-refresh/pull/1746 that guided this.
